### PR TITLE
fix(qwik-nx): update nxkit to v2.1.1

### LINKED ISSUE #44: creating new nx workspace with preset=qwik-nx using pnpx throws an error
**Describe the bug**

Creating a new workspace with the qwik-nx preset throws (I'm using windows 10)

**To Reproduce**

Run ` pnpx create-nx-workspace --preset qwik-nx`

**Expected behavior**
To complete with no errors

**Screenshots**
```
node:events:491
      throw er; // Unhandled 'error' event
      ^

Error: spawn pnpm ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:283:19)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
Emitted 'error' event on ChildProcess instance at:
    at ChildProcess._handle.onexit (node:internal/child_process:289:12)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn pnpm',
  path: 'pnpm',
  spawnargs: [
    'exec',
    'nx',
    '--nxWorkspaceRoot="C:/dev/workspace/open-source"',
    'g',
    'qwik-nx:preset',
    '--no-a',
    '--no-g',
    '--$0=C:\\Users\\MyUser\\AppData\\Local\\pnpm\\store\\v3\\tmp\\dlx-19560\\node_modules\\create-nx-workspace\\bin\\index.js',
    '--packageManager=pnpm',
    '--defaultBase=main'
  ]
}

Node.js v18.7.0
 ERROR  Command failed with exit code 1: C:\Users\MyUser\AppData\Local\pnpm\store\v3\tmp\dlx-19560\node_modules\.bin\create-nx-workspace.CMD --preset qwik-nx

pnpm: Command failed with exit code 1: C:\Users\MyUser\AppData\Local\pnpm\store\v3\tmp\dlx-19560\node_modules\.bin\create-nx-workspace.CMD --preset qwik-nx
    at makeError (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:22852:17)
    at handlePromise (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:23423:33)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.handler [as dlx] (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:202440:7)
    at async C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:209882:21
    at async main (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:209853:34)
    at async runPnpm (C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:210079:5)
    at async C:\Users\MyUser\AppData\Roaming\nvm\v18.7.0\node_modules\pnpm\dist\pnpm.cjs:210071:7

```

**Desktop (please complete the following information):**

- OS: Windows 10

### PR DESCRIPTION
**Pull request by sebastiandg7**
# What is it?

- [ ] Feature / enhancement
- [x] Bug
- [ ] Docs / tests

# Description

Updates NxKit to v2.1.1 to include fix in nxkit/nxkit#59 allowing to install the packages dynamically.

# Use cases and why

- 1. Context in #44

# Screenshots/Demo

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-nx/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

Fixes #44

### CODE REVIEW

File: packages/qwik-nx/src/utils/versions.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const postcssVersion = '8.4.16';
 export const tailwindcssVersion = '3.1.8';
 
 // nxkit packages
-export const nxKitVersion = '^2.0.1';
+export const nxKitVersion = '2.1.1';
```
[the remark this answers is not in the document]
Comment (dmitry-stepanenko) [reply packages/qwik-nx/src/utils/versions.ts:20]: And probably you have to also bump it in package.json

[the remark this answers is not in the document]
Comment (sebastiandg7) [reply packages/qwik-nx/src/utils/versions.ts:20]: Got that!

File: tsconfig.base.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
-      "qwik-nx": ["packages/qwik-nx/src/index.ts"],
-      "@qwikifiers/e2e/utils": ["e2e/utils"]
```
Comment (sebastiandg7) [inline tsconfig.base.json:19]: `pnpm nx format` did it 🤠

**Comment by sebastiandg7** [comment]
Good to go now @dmitry-stepanenko!

File: tsconfig.base.json (hunk 2 above)
Comment (dmitry-stepanenko) [reply tsconfig.base.json:19]: Yeah, not sure how this sneaker in, we have format in pre-commit 😅

**Review by dmitry-stepanenko — APPROVED** [review]
Thanks @sebastiandg7 !

**Pull request merged**